package.json

```diff
@@ -31,19 +31,22 @@
     "@actions/core": "^1.10.0",
     "@actions/exec": "^1.0.1",
     "@actions/github": "^2.2.0",
-    "@actions/io": "^1.0.1",
+    "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^1.1.2",
+    "minimatch": "^5.1.1",
+    "npm": "^8.19.2",
```

Comment:
I'm curious if we need `npm` as a dependency here? could we use an `engine` object inside of the file instead? something like this
```json
"engines": {
    "node": ">=12",
    "npm": ">=6"
  },
```

Reply:
@vmjoseph could you help with this? `npm` was added as a dep in one of your commits and not sure the reason.